packages/checkout/widgets-lib/src/widgets/wallet/components/NetworkMenu/NetworkMenu.tsx

```diff
@@ -65,8 +65,6 @@ export function NetworkMenu({ setBalancesLoading }: NetworkMenuProps) {
       });
 
       try {
-        setBalancesLoading(true);
```

Comment:
is the `setBalancesLoading(false);` needed in the catch below still?

Reply:
No good catch, I've removed it.